src/main/java/frc/robot/RobotContainer.java

```diff
@@ -1,48 +1,146 @@
- * periodic methods (other than the scheduler calls). Instead, the structure of the robot (including
- * subsystems, commands, and button mappings) should be declared here.
+ * This class is where the bulk of the robot should be declared. Since
+ * Command-based is a "declarative" paradigm, very little robot logic should
+ * actually be handled in the {@link Robot} periodic methods (other than the
+ * scheduler calls). Instead, the structure of the robot (including subsystems,
+ * commands, and button mappings) should be declared here.
  */
 public class RobotContainer {
-  // The robot's subsystems and commands are defined here...
-  private final ExampleSubsystem m_exampleSubsystem = new ExampleSubsystem();
-
-  private final ExampleCommand m_autoCommand = new ExampleCommand(m_exampleSubsystem);
-
-  /** The container for the robot. Contains subsystems, OI devices, and commands. */
-  public RobotContainer() {
-    // Configure the button bindings
-    configureButtonBindings();
-  }
-
-  /**
-   * Use this method to define your button->command mappings. Buttons can be created by
-   * instantiating a {@link GenericHID} or one of its subclasses ({@link
-   * edu.wpi.first.wpilibj.Joystick} or {@link XboxController}), and then passing it to a {@link
-   * edu.wpi.first.wpilibj2.command.button.JoystickButton}.
-   */
-  private void configureButtonBindings() {}
-
-  /**
-   * Use this to pass the autonomous command to the main {@link Robot} class.
-   *
-   * @return the command to run in autonomous
-   */
-  public Command getAutonomousCommand() {
-    // An ExampleCommand will run in autonomous
-    return m_autoCommand;
-  }
+	// The robot's subsystems and commands are defined here...
+
+	public final DriveSubsystem driveSubsystem = new DriveSubsystem();
+	// public final ServoSubsystem servoSubsystem = new ServoSubsystem();
+
+	/**
+	 * The container for the robot. Contains subsystems, OI devices, and commands.
+	 * 
+	 */
+
+	// Tank Drive
+	private XboxController driver;
+	private Joystick leftJoystick;
+	private Joystick rightJoystick;
+	private JoystickButton buttonA;
+	private JoystickButton buttonB;
+	private JoystickButton buttonX;
+	private JoystickButton buttonY;
+
+	// Arcade Drive
+	private Joystick wheel;
+	private Joystick throttle;
+	private Joystick buttonPanel;
+	public RobotContainer() {
+		// Configure the button bindings
+		configureButtonBindings();
+	}
+
+	/**
+	 * Use this method to define your button->command mappings. Buttons can be
+	 * created by instantiating a {@link GenericHID} or one of its subclasses
+	 * ({@link edu.wpi.first.wpilibj.Joystick} or {@link XboxController}), and then
+	 * passing it to a {@link edu.wpi.first.wpilibj2.command.button.JoystickButton}.
+	 */
+	private void configureButtonBindings() {
+		driver = new XboxController(Constants.XBOX_CONTROLLER_PORT);
+		// Tank Drive
+		leftJoystick = new Joystick(Constants.LEFT_JOYSTICK_PORT);
```

Comment:
I think we only create one Joystick per controller. The port number corresponds to the Xbox controller number.